cfpb/cfgov-refresh
# Fixes for form explainer pages

Fixes for form explainer pages

## Changes

- Modified code to fix the following issues:
  - fixes issue with using `classlist.repace` on Internet Explorer.
  - fixes issue with anchor link focus not scrolling on IOS.
  - fixes issue with page scrolling on initial load.
  - fixes issue with the `Skip to main content` button not working.

## Testing

1.  Visit `http://localhost:8000/owning-a-home/closing-disclosure/`.
2.  Click the tab button.
3.  Click the `Skip to main content` and verify it scrolls to the proper place.
4.  Visit the page using IE10. 
5.  Click the expandable and verify there are no errors in the console.
6.  Visit the page using and Iphone emulator ( Xcode or Sauce )
7.  Click on the form and verify the expandable is scrolled into view.

## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
